File: app/src/main/java/com/noodle/sample/ui/MainActivity.java
package com.noodle.sample.ui;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.view.View;

import com.noodle.sample.R;

public class MainActivity extends AppCompatActivity {

  @Override
  protected void onCreate(@Nullable final Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);

    findViewById(R.id.addBook).setOnClickListener(new View.OnClickListener() {
      @Override
      public void onClick(final View v) {
        final AddBookFragment addBookFragment = new AddBookFragment();
        addBookFragment.show(getSupportFragmentManager(), "add_book");
      }
    });
  }

}

File: app/src/main/java/com/noodle/sample/ui/MainActivity.kt
package com.noodle.sample.ui

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.noodle.sample.R

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setTitle(R.string.main_title)

        findViewById<View>(R.id.addBook).setOnClickListener {
            val editBookFragment = EditBookFragment()
            editBookFragment.show(supportFragmentManager, "add_book")
        }
    }

}
